### PR TITLE
data ingest review

### PR DESCRIPTION
**Pull request by kusweta**
Few modifications in the data-ingest ADR

Please ignore the adr - data-schema-strategy. It's still in progress.

### CODE REVIEW

File: data-reconciliation-app/docs/adr/01-data-ingest.md [hunk 1]
```diff
@@ -204,7 +202,7 @@ An E2E data flow is captured [here](../images/data_recon_sample.png) with input
 
 ### 4. K-V Design - Expensive to Reconcile & Report
 
-By only storing the `ReconciledRecord`, we make "updating" of a key for a particular member harder. Addititionally, when we create a report on the reconciled data, members/users will only receive a report on the keys they submitted. By using the `ReconciledRecord`, we will have to check the values for each key to create the report.
+By only storing the `ReconciledRecord`, we made key updation for a particular member harder. Additionally, when we create a report on the reconciled data, members/users will only receive a report on the keys they submitted. By using the `ReconciledRecord`, we will have to check the values for each key to create the report.
```
Comment (mkcomer) [inline data-reconciliation-app/docs/adr/01-data-ingest.md:205]: Do we want to explain why it is harder/expensive? 

"Additionally, when we create a report on the reconciled data, members/users will only receive a report on the keys they submitted" - I feel like this is a requirement/irrelevant to this point?

Comment (mkcomer) [reply data-reconciliation-app/docs/adr/01-data-ingest.md:205]: Nit: Is updation a word? 

Maybe - "By only storing the `ReconciledRecord`, updating a key for a particular member is more expensive"

**Comment by mkcomer** [comment]
@kusweta small nits. Can we also run this doc/all docs through spell check? Spotted a couple misspelled words. Thanks!

File: data-reconciliation-app/docs/adr/01-data-ingest.md (hunk 1 above)
Comment (kusweta) [reply data-reconciliation-app/docs/adr/01-data-ingest.md]: Acknowledged!  
Updated the ADR

**Review by mkcomer — APPROVED** [review]
Thanks for updating. LGTM

**Review by julioalex-rezende — APPROVED** [review]

File: data-reconciliation-app/docs/adr/01-data-ingest.md [hunk 2]
```diff
@@ -6,17 +6,17 @@ Approved
 
 ## Context
 
-We need to build an API that ingests data into our system. Once ingested, we need to store this data in the K-V store so we can reconcile and report out on the data.
+We need to build an API that ingests data into our system. Once ingested, we need to store this data in the K-V store so we can reconcile it and get reports on the ingested data.
```
Comment (mkcomer) [inline data-reconciliation-app/docs/adr/01-data-ingest.md:9]: nit: and `generate` reports on the ingested data

File: data-reconciliation-app/docs/adr/01-data-ingest.md [hunk 3]
```diff
@@ -6,17 +6,17 @@ Approved
...
 
 Gathered requirements:
 
 - Members of the network will decide on a data schema ahead of ingest.
-- Members and Users of the network can ingest data via the API. Data will be ingested via JSON which will include:
+- Members and users of the network can ingest data via the API. Data will be ingested via JSON, which will include:
```
Comment (mkcomer) [inline data-reconciliation-app/docs/adr/01-data-ingest.md:14]: we can also ingest via csv is that correct?

Comment (kusweta) [reply data-reconciliation-app/docs/adr/01-data-ingest.md]: yes, have updated it in the ADR.

**Pull request merged**